src/main/modules/mpris.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * Module in charge of setting and manage mpris service for linux
+   TODO: fetch cover, handle seek bar, handle volume    
+ */
+import * as electron from 'electron';
+import ModuleWindow from './module-window';
+import { TrackModel } from '../../shared/types/interfaces';
+import { fetchCover } from '../../shared/utils';
+import * as mpris  from 'mpris-service';
+const { app, ipcMain } = electron;
+
+class MprisModule extends ModuleWindow {
+
+  player = mpris({
+      name: 'museeks',
+      identity: 'Musseks Music Player',
+      canRaise: true,
+      supportedUriSchemes: ['file'],
+      supportedMimeTypes: ['audio/mpeg', 'application/ogg'],
+      supportedInterfaces: ['player', 'trackList']
+  });
+
+  constructor (window: Electron.BrowserWindow) {
+    super(window);
+    this.platforms = ['linux'];
+  }
+
+	async load() {
```

Comment:
you have an indentation problem. You should catch it by running `npm run lint:ts`

Reply:
The IDE didn't show me anything :confused:

Reply:
I didn't have installed the ts linter. Sorry. It's solved xD